app/server.ts

```diff
@@ -0,0 +1,41 @@
+import * as Express from 'express';
+import * as BodyParser from 'body-parser';
+import * as Morgan from 'morgan';
+
+import { PORT } from './modules/util/config';
+import { BasePath as ProjectBasePath, projectRouter } from './modules/project';
+
+export class Server {
+    server: Express.Application;
+    port: string;
+
+    constructor() {
+        this.server = Express();
+        this.port = PORT;
+        this.config();
+        this.defaultRoute();
+    }
+
+    config = () => {
```

Comment:
You tend to love Arrow functions a lot. Just to make a shout out that they come with their own cost 😉

Reply:
I tried reading about 'this'. But not completely clear. Would need your help to understand better.